firmware/application/apps/ui_looking_glass_app.cpp

```diff
@@ -518,7 +518,7 @@ GlassView::GlassView(
 }
 
 void GlassView::load_Presets() {
```

Comment:
This has nothing to do with your PR but I always thought looking glass should use a freqman preset instead of it's own file format. 
The plan was:
-convert actual preset to a freqman file (really nopt hard), set it as default preset
-add an 'open' button to allow changing preset file
I may do this one day.